### ISSUE TITLE
Scroll to fast since last update

### ISSUE THREAD

**Issue by noahfraiture** [hyprwm/Hyprland#6496]
### Bug or Regression?

Bug

### System Info and Version

```
Hyprland, built from branch  at commit ea2501d4556f84d3de86a4ae2f4b22a474555b9f  (props: bump version to 0.41.0).
Date: Mon Jun 10 14:23:06 2024
Tag: v0.41.0, commits: 4794

flags: (if any)


System Information:
System name: Linux
Node name: ni16ps7
Release: 6.9.4-arch1-1
Version: #1 SMP PREEMPT_DYNAMIC Wed, 12 Jun 2024 20:17:17 +0000


GPU information: 
0000:00:02.0 VGA compatible controller [0300]: Intel Corporation Raptor Lake-P [Iris Xe Graphics] [8086:a7a0] (rev 04) (prog-if 00 [VGA controller])
0000:01:00.0 VGA compatible controller [0300]: NVIDIA Corporation AD107M [GeForce RTX 4060 Max-Q / Mobile] [10de:28a0] (rev a1) (prog-if 00 [VGA controller])


os-release: NAME="EndeavourOS"
PRETTY_NAME="EndeavourOS"
ID="endeavouros"
ID_LIKE="arch"
BUILD_ID="2023.11.17"
ANSI_COLOR="38;2;23;147;209"
HOME_URL="https://endeavouros.com"
DOCUMENTATION_URL="https://discovery.endeavouros.com"
SUPPORT_URL="https://forum.endeavouros.com"
BUG_REPORT_URL="https://forum.endeavouros.com/c/general-system/endeavouros-installation"
PRIVACY_POLICY_URL="https://endeavouros.com/privacy-policy-2"
LOGO="endeavouros"


plugins:

======Config-Start======


Config File: /home/noah/.config/hypr/hyprland.conf: Read Succeeded
# #####################################################################################
# AUTOGENERATED HYPR CONFIG.
# PLEASE USE THE CONFIG PROVIDED IN THE GIT REPO /examples/hypr.conf AND EDIT IT,
# OR EDIT THIS ONE ACCORDING TO THE WIKI INSTRUCTIONS.
# #######################################################################################

# Please note not all available settings / options are set here.
# For a full list, see the wiki
# autogenerated = 1 # remove this line to remove the warning



# █▀▄▀█ █▀█ █▄░█ █ ▀█▀ █▀█ █▀█
# █░▀░█ █▄█ █░▀█ █ ░█░ █▄█ █▀▄

# See https://wiki.hyprland.org/Configuring/Monitors/

monitor = eDP-1, 2560x1600@120, 0x0, 1# mirror, HDMI-A-1
monitor = HDMI-A-1, 1920x1080, 2560x0,1

# unscale XWayland
xwayland {
  force_zero_scaling = true
}



# █░░ ▄▀█ █░█ █▄░█ █▀▀ █░█
# █▄▄ █▀█ █▄█ █░▀█ █▄▄ █▀█

# See https://wiki.hyprland.org/Configuring/Keywords/ for more
# Execute your favorite apps at launch

$scrPath = ~/.config/hypr/scripts
exec-once = $scrPath/resetxdgportal.sh # reset XDPH for screenshare
exec-once = dbus-update-activation-environment --systemd WAYLAND_DISPLAY XDG_CURRENT_DESKTOP # for XDPH
exec-once = dbus-update-activation-environment --systemd --all # for XDPH
exec-once = systemctl --user import-environment WAYLAND_DISPLAY XDG_CURRENT_DESKTOP # for XDPH
exec-once = /usr/lib/polkit-kde-authentication-agent-1 # authentication dialogue for GUI apps
exec-once = waybar # launch the system panel
exec-once = blueman-applet # systray app for BT
exec-once = nm-applet --indicator # systray app for Network/Wifi
exec-once = dunst # start notification demon
exec-once = wl-paste --type text --watch cliphist store # clipboard store text data
exec-once = wl-paste --type image --watch cliphist store # clipboard store image data
exec-once = $scrPath/swwwallpaper.sh # start wallpaper daemon
exec-once = $scrPath/batterynotify.sh # battery notification
exec-once = emote



# █▀▀ █▄░█ █░█
# ██▄ █░▀█ ▀▄▀

# Some default env vars.

env = XDG_CURRENT_DESKTOP,Hyprland
env = XDG_SESSION_TYPE,wayland
env = XDG_SESSION_DESKTOP,Hyprland
env = GDK_BACKEND,wayland
env = QT_QPA_PLATFORM,wayland
#env = QT_STYLE_OVERRIDE,kvantum
env = QT_QPA_PLATFORMTHEME,qt5ct
env = QT_WAYLAND_DISABLE_WINDOWDECORATION,1
env = QT_AUTO_SCREEN_SCALE_FACTOR,1
# env = MOZ_ENABLE_WAYLAND,1 # Not present
# env = GDK_SCALE,1



# █ █▄░█ █▀█ █░█ ▀█▀
# █ █░▀█ █▀▀ █▄█ ░█░

# For all categories, see https://wiki.hyprland.org/Configuring/Variables/

input {
    kb_layout = us
    kb_variant = altgr-intl
    follow_mouse = 1


    touchpad {
        natural_scroll = yes
        scroll_factor = 0.7
        disable_while_typing = false
    }

    sensitivity = 0 # -1.0 - 1.0, 0 means no modification.
    force_no_accel = false

    kb_options = caps:swapescape
}

# See https://wiki.hyprland.org/Configuring/Variables/ for more

gestures {
    workspace_swipe = true
    workspace_swipe_fingers = 3
}



# █░░ ▄▀█ █▄█ █▀█ █░█ ▀█▀ █▀
# █▄▄ █▀█ ░█░ █▄█ █▄█ ░█░ ▄█

# See https://wiki.hyprland.org/Configuring/Dwindle-Layout/ for more

dwindle {
    pseudotile = yes # master switch for pseudotiling. Enabling is bound to mainMod + P in the keybinds section below
    preserve_split = yes # you probably want this
    smart_split=true
    no_gaps_when_only=1
}

# See https://wiki.hyprland.org/Configuring/Master-Layout/ for more

master {
    new_is_master = true
}



# █▀▄▀█ █ █▀ █▀▀
# █░▀░█ █ ▄█ █▄▄

# See https://wiki.hyprland.org/Configuring/Variables/ for more

misc {
    vrr = 0
    disable_hyprland_logo = true
}

$LAPTOP_KB_ENABLED = true
device {
  name = at-translated-set-2-keyboard
  enabled = $LAPTOP_KB_ENABLED
}


# █▀ █▀█ █░█ █▀█ █▀▀ █▀▀
# ▄█ █▄█ █▄█ █▀▄ █▄▄ ██▄

# Source a file (multi-file configs)
# source = ~/.config/hypr/myColors.conf

source = ~/.config/hypr/animations.conf
source = ~/.config/hypr/keybindings.conf
source = ~/.config/hypr/windowrules.conf
source = ~/.config/hypr/themes/common.conf # shared theme settings
source = ~/.config/hypr/themes/theme.conf # theme specific settings
source = ~/.config/hypr/themes/colors.conf # wallbash color override
source = ~/.config/hypr/monitors.conf # initially empty, to be configured by user and remains static
source = ~/.config/hypr/userprefs.conf # initially empty, to be configured by user and remains static
# source = ~/.config/hypr/hyprlock.conf

# Note: as userprefs.conf is sourced at the end, settings configured in this file will override the defaults

source = ~/.config/hypr/nvidia.conf # auto sourced vars for nvidia



Config File: /home/noah/.config/hypr/animations.conf: Read Succeeded

# ▄▀█ █▄░█ █ █▀▄▀█ ▄▀█ ▀█▀ █ █▀█ █▄░█
# █▀█ █░▀█ █ █░▀░█ █▀█ ░█░ █ █▄█ █░▀█


# Some default animations, see https://wiki.hyprland.org/Configuring/Animations/ for more

animations {
    enabled = yes
    bezier = wind, 0.05, 0.9, 0.1, 1.05
    bezier = winIn, 0.1, 1.1, 0.1, 1.1
    bezier = winOut, 0.3, -0.3, 0, 1
    bezier = liner, 1, 1, 1, 1
    animation = windows, 1, 6, wind, slide
    animation = windowsIn, 1, 6, winIn, slide
    animation = windowsOut, 1, 5, winOut, slide
    animation = windowsMove, 1, 5, wind, slide
    animation = border, 1, 1, liner
    # animation = borderangle, 1, 30, liner, loop # breathing
    animation = fade, 1, 10, default
    animation = workspaces, 1, 5, wind
}



Config File: /home/noah/.config/hypr/keybindings.conf: Read Succeeded

# █▄▀ █▀▀ █▄█ █▄▄ █ █▄░█ █▀▄ █ █▄░█ █▀▀ █▀
# █░█ ██▄ ░█░ █▄█ █ █░▀█ █▄▀ █ █░▀█ █▄█ ▄█


# See https://wiki.hyprland.org/Configuring/Keywords/ for more
# Example binds, see https://wiki.hyprland.org/Configuring/Binds/ for more

# Main modifier
$mainMod = SUPER_L # windows key

# Disable keybind
bind = $mainMod, C, submap, clean
submap = clean
bind = $mainMod, C, submap, reset
submap = reset

# Window/Session actions
$srcPath = ~/.config/hypr/scripts
bind = $mainMod, Q, exec, $srcPath/dontkillsteam.sh # killactive, kill the window on focus
bind = $mainMod, delete, exit, # kill hyperland session
bind = $mainMod, W, togglefloating, # toggle the window on focus to float
bind = $mainMod, F, fullscreen, # toggle the window on focus to fullscreen
bind = $mainMod, Z, exec, $srcPath/gamemode.sh # zen mode
bind = $mainMod, escape, exec, $srcPath/logoutlaunch.sh 1 # logout menu
bind = $mainMod, B, toggleopaque # save battery
bind = $mainMod, P, pin
bind = $mainMod, G, togglegroup, # toggle the window on focus to float
bind = $mainMod, escape, exec, $srcPath/logoutlaunch.sh 1 # logout menu
bind = CONTROL, ESCAPE, exec, killall waybar || waybar # toggle waybar

# Exec custom scripts
bind = $mainMod SHIFT, D, exec, $srcPath/wallbashtoggle.sh  # toggle wallbash on/off
bind = $mainMod SHIFT, T, exec, pkill -x rofi || $srcPath/themeselect.sh # theme select menu
bind = $mainMod SHIFT, A, exec, pkill -x rofi || $srcPath/rofiselect.sh # rofi style select menu
bind = $mainMod SHIFT, W, exec, pkill -x rofi || $srcPath/swwwallselect.sh # rofi wall select menu
bind = , XF86LaunchA, exec, pkill -x rofi || $srcPath/swwwallselect.sh # rofi wall select menu
bind = , XF86LaunchB, exec, pkill -x rofi || $srcPath/themeselect.sh # theme select menu

# Application shortcuts
bind = $mainMod, T, exec, kitty  # open terminal
bind = $mainMod CTRL, T, exec, kitty --class floating
bind = $mainMod, E, exec, kitty --class floating -e nnn
bind = $mainMod, O, exec, opera
bind = $mainMod, V, exec, $srcPath/cliphist.sh c 2
bind = $mainMod, V, exec, pkill -x rofi || $srcPath/cliphist.sh c  # open Pasteboard in screen center
# bind = $mainMod, V, exec, kitty --class floating -e zsh -c 'clipse $PPID'
bind = $mainMod, R, exec, ferdium

# Rofi is toggled on/off if you repeat the key presses
bind = $mainMod, A, exec, pkill -x rofi || $srcPath/rofilaunch.sh d # launch desktop applications

# Audio control
bind = , XF86AudioMute, exec, $srcPath/volumecontrol.sh -o m # toggle audio mute
bind = , XF86AudioMicMute, exec, $srcPath/volumecontrol.sh -i m # toggle microphone mute
bind = , XF86AudioLowerVolume, exec, $srcPath/volumecontrol.sh -o d # decrease volume
bind = , XF86AudioRaiseVolume, exec, $srcPath/volumecontrol.sh -o i # increase volume
bind = , XF86AudioPlay, exec, playerctl play-pause
bind = , XF86AudioPause, exec, playerctl play-pause
bind = , XF86AudioNext, exec, playerctl next
bind = , XF86AudioPrev, exec, playerctl previous

# Brightness control
bindel = , XF86MonBrightnessUp, exec, $scrPath/brightnesscontrol.sh i # increase brightness
bindel = , XF86MonBrightnessDown, exec, $scrPath/brightnesscontrol.sh d # decrease brightness

# Screenshot/Screencapture
# bind = , Print, exec, $srcPath/screenshot.sh sf # screenshot snip
# bind = $mainMod, Print, exec, $srcPath/screenshot.sh p # print current screen
bind = $mainMod SHIFT, 4, exec, $srcPath/screenshot.sh sf


# Move focus with mainMod + arrow keys
bind = $mainMod, h, movefocus, l
bind = $mainMod, l, movefocus, r
bind = $mainMod, k, movefocus, u
bind = $mainMod, j, movefocus, d
bind = ALT, Tab, movefocus, d

# Switch workspaces with mainMod + [0-9]
bind = $mainMod, 1, workspace, 1
bind = $mainMod, 2, workspace, 2
bind = $mainMod, 3, workspace, 3
bind = $mainMod, 4, workspace, 4
bind = $mainMod, 5, workspace, 5
bind = $mainMod, 6, workspace, 6
bind = $mainMod, 7, workspace, 7
bind = $mainMod, 8, workspace, 8
bind = $mainMod, 9, workspace, 9
bind = $mainMod, 0, workspace, 10

# Switch workspaces relative to the active workspace with mainMod + CTRL + [←→]
bind = $mainMod CTRL, l, workspace, r+1 
bind = $mainMod CTRL, h, workspace, r-1

# move to the first empty workspace instantly with mainMod + CTRL + [↓]
bind = $mainMod CTRL, j, workspace, empty 

# Move active window to a workspace with mainMod + SHIFT + [0-9]
bind = $mainMod SHIFT, 1, movetoworkspace, 1
bind = $mainMod SHIFT, 2, movetoworkspace, 2
bind = $mainMod SHIFT, 3, movetoworkspace, 3
# bind = $mainMod SHIFT, 4, movetoworkspace, 4 
# replace by screenshot for keychron mac keybinds

bind = $mainMod SHIFT, 5, movetoworkspace, 5
bind = $mainMod SHIFT, 6, movetoworkspace, 6
bind = $mainMod SHIFT, 7, movetoworkspace, 7
bind = $mainMod SHIFT, 8, movetoworkspace, 8
bind = $mainMod SHIFT, 9, movetoworkspace, 9
bind = $mainMod SHIFT, 0, movetoworkspace, 10

bind = $mainMod, TAB, workspace,e+1

# Move active window to a relative workspace with mainMod + CTRL + ALT + [←→]
bind = $mainMod SHIFT, l, movetoworkspace, r+1
bind = $mainMod SHIFT, h, movetoworkspace, r-1

# Move active window around current workspace with mainMod + SHIFT + CTRL [←→↑↓]
bind = $mainMod SHIFT $CONTROL, h, movewindow, l
bind = $mainMod SHIFT $CONTROL, l, movewindow, r
bind = $mainMod SHIFT $CONTROL, k, movewindow, u
bind = $mainMod SHIFT $CONTROL, j, movewindow, d

# Move/Resize windows with mainMod + LMB/RMB and dragging
bindm = $mainMod, mouse:272, movewindow
bindm = $mainMod, mouse:273, resizewindow

# Special workspaces (scratchpad)
bind = $mainMod CTRL, S, movetoworkspacesilent, special
bind = $mainMod, S, togglespecialworkspace,

# Toggle Layout
bind = $mainMod, M, togglesplit, # dwindle

# Move window silently to workspace Super + Alt + [0-9]
bind = $mainMod CTRL, 1, movetoworkspacesilent, 1
bind = $mainMod CTRL, 2, movetoworkspacesilent, 2
bind = $mainMod CTRL, 3, movetoworkspacesilent, 3
bind = $mainMod CTRL, 4, movetoworkspacesilent, 4
bind = $mainMod CTRL, 5, movetoworkspacesilent, 5
bind = $mainMod CTRL, 6, movetoworkspacesilent, 6
bind = $mainMod CTRL, 7, movetoworkspacesilent, 7
bind = $mainMod CTRL, 8, movetoworkspacesilent, 8
bind = $mainMod CTRL, 9, movetoworkspacesilent, 9
bind = $mainMod CTRL, 0, movetoworkspacesilent, 10

# Trigger when the switch is turning off
bindl= , switch:on:Lid Switch, exec, swaylock && systemctl suspend



Config File: /home/noah/.config/hypr/windowrules.conf: Read Succeeded

# █░█░█ █ █▄░█ █▀▄ █▀█ █░█░█   █▀█ █░█ █░░ █▀▀ █▀
# ▀▄▀▄▀ █ █░▀█ █▄▀ █▄█ ▀▄▀▄▀   █▀▄ █▄█ █▄▄ ██▄ ▄█


# Example windowrule v1
# windowrule = float, ^(kitty)$
# Example windowrule v2
# windowrulev2 = float,class:^(kitty)$,title:^(kitty)$
# See https://wiki.hyprland.org/Configuring/Window-Rules/ for more

windowrulev2 = opacity 1 1,class:^(firefox)$
windowrulev2 = opacity 1.00 1.00,class:^(opera)$
windowrulev2 = opacity 0.90 0.90,class:^(Brave-browser)$
windowrulev2 = opacity 1 1,class:^(Steam)$
windowrulev2 = opacity 1 1,class:^(steam)$
windowrulev2 = opacity 0.80 0.80,class:^(steamwebhelper)$
windowrulev2 = opacity 1.00 1.00,class:^(Spotify)$
windowrulev2 = opacity 0.80 0.80,class:^(Code)$
windowrulev2 = opacity 0.80 0.80,class:^(code-url-handler)$
windowrulev2 = opacity 0.80 0.80,class:^(kitty)$
windowrulev2 = opacity 0.80 0.80,class:^(org.kde.dolphin)$
windowrulev2 = opacity 0.80 0.80,class:^(org.kde.ark)$
windowrulev2 = opacity 0.80 0.80,class:^(nwg-look)$
windowrulev2 = opacity 0.80 0.80,class:^(qt5ct)$

windowrulev2 = opacity 0.90 0.90,class:^(com.github.rafostar.Clapper)$ #Clapper-Gtk
windowrulev2 = opacity 0.80 0.80,class:^(com.github.tchx84.Flatseal)$ #Flatseal-Gtk
windowrulev2 = opacity 0.80 0.80,class:^(hu.kramo.Cartridges)$ #Cartridges-Gtk
windowrulev2 = opacity 0.80 0.80,class:^(com.obsproject.Studio)$ #Obs-Qt
windowrulev2 = opacity 0.80 0.80,class:^(gnome-boxes)$ #Boxes-Gtk
windowrulev2 = opacity 0.80 0.80,class:^(discord)$ #Discord-Electron
windowrulev2 = opacity 0.80 0.80,class:^(WebCord)$ #WebCord-Electron
windowrulev2 = opacity 0.80 0.80,class:^(app.drey.Warp)$ #Warp-Gtk
windowrulev2 = opacity 0.80 0.80,class:^(net.davidotek.pupgui2)$ #ProtonUp-Qt
windowrulev2 = opacity 0.80 0.80,class:^(yad)$ #Protontricks-Gtk
windowrulev2 = opacity 0.80 0.80,class:^(Signal)$ #Signal-Gtk
windowrulev2 = opacity 0.80 0.80,class:^(io.gitlab.theevilskeleton.Upscaler)$ #Upscaler-Gtk

windowrulev2 = opacity 0.80 0.70,class:^(pavucontrol)$
windowrulev2 = opacity 0.80 0.70,class:^(blueman-manager)$
windowrulev2 = opacity 0.80 0.70,class:^(nm-applet)$
windowrulev2 = opacity 0.80 0.70,class:^(nm-connection-editor)$
windowrulev2 = opacity 0.80 0.70,class:^(org.kde.polkit-kde-authentication-agent-1)$

windowrulev2 = float,class:^(qt5ct)$
windowrulev2 = float,class:^(nwg-look)$
windowrulev2 = float,class:^(org.kde.ark)$
windowrulev2 = float,class:^(Signal)$ #Signal-Gtk
windowrulev2 = float,class:^(com.github.rafostar.Clapper)$ #Clapper-Gtk
windowrulev2 = float,class:^(app.drey.Warp)$ #Warp-Gtk
windowrulev2 = float,class:^(net.davidotek.pupgui2)$ #ProtonUp-Qt
windowrulev2 = float,class:^(yad)$ #Protontricks-Gtk
windowrulev2 = float,class:^(eog)$ #Imageviewer-Gtk
windowrulev2 = float,class:^(io.gitlab.theevilskeleton.Upscaler)$ #Upscaler-Gtk
windowrulev2 = float,class:^(pavucontrol)$
windowrulev2 = float,class:^(blueman-manager)$
windowrulev2 = float,class:^(nm-applet)$
windowrulev2 = float,class:^(nm-connection-editor)$
windowrulev2 = float,class:^(org.kde.polkit-kde-authentication-agent-1)$

# overlay poker
windowrulev2 = opacity 1 1,title:^(overlay)$ # can remove border
# windowrulev2 = noblur,title:^(overlay)$
windowrulev2 = float,title:^(overlay)$

# Floating terminal
windowrulev2 = opacity 0.80 0.80,class:^(floating)$
windowrulev2 = float,class:^(floating)$
windowrulev2 = move cursor -50% -50%,class:^(floating)$
windowrulev2 = size 25% 30%,class:^(floating)$

workspace = special, gapsin:5, gapsout:100


Config File: /home/noah/.config/hypr/themes/common.conf: Read Succeeded

#█▀▀ █░█ █▀█ █▀ █▀█ █▀█
#█▄▄ █▄█ █▀▄ ▄█ █▄█ █▀▄

#env = XCURSOR_THEME,Bibata-Modern-Ice
#env = XCURSOR_SIZE,20

exec = hyprctl setcursor Bibata-Modern-Ice 20
exec = gsettings set org.gnome.desktop.interface cursor-theme 'Bibata-Modern-Ice'
exec = gsettings set org.gnome.desktop.interface cursor-size 20


#█▀▀ █▀█ █▄░█ ▀█▀
#█▀░ █▄█ █░▀█ ░█░

exec = gsettings set org.gnome.desktop.interface font-name 'Cantarell 10'
exec = gsettings set org.gnome.desktop.interface document-font-name 'Cantarell 10'
exec = gsettings set org.gnome.desktop.interface monospace-font-name 'CaskaydiaCove Nerd Font Mono 9'
exec = gsettings set org.gnome.desktop.interface font-antialiasing 'rgba'
exec = gsettings set org.gnome.desktop.interface font-hinting 'full'


#█▀ █▀█ █▀▀ █▀▀ █ ▄▀█ █░░
#▄█ █▀▀ ██▄ █▄▄ █ █▀█ █▄▄

decoration {
    dim_special = 0.3
    blur {
        special = true
    }
}



Config File: /home/noah/.config/hypr/themes/theme.conf: Read Succeeded
exec = gsettings set org.gnome.desktop.interface icon-theme 'Tela-circle-dracula'
exec = gsettings set org.gnome.desktop.interface gtk-theme 'Catppuccin-Mocha'
exec = gsettings set org.gnome.desktop.interface color-scheme 'prefer-dark'

general {
    gaps_in = 3
    gaps_out = 8
    border_size = 2
    col.active_border = rgba(ca9ee6ff) rgba(f2d5cfff) 45deg
    col.inactive_border = rgba(b4befecc) rgba(6c7086cc) 45deg
    layout = dwindle
    resize_on_border = true
}

group {
    col.border_active = rgba(ca9ee6ff) rgba(f2d5cfff) 45deg
    col.border_inactive = rgba(b4befecc) rgba(6c7086cc) 45deg
    col.border_locked_active = rgba(ca9ee6ff) rgba(f2d5cfff) 45deg
    col.border_locked_inactive = rgba(b4befecc) rgba(6c7086cc) 45deg
}

decoration {
    rounding = 10
    drop_shadow = false

    blur {
        enabled = yes
        size = 6
        passes = 3
        new_optimizations = on
        ignore_opacity = on
        xray = false
    }
}

layerrule = blur,waybar


Config File: /home/noah/.config/hypr/themes/colors.conf: Read Succeeded


Config File: /home/noah/.config/hypr/monitors.conf: Read Succeeded


Config File: /home/noah/.config/hypr/userprefs.conf: Read Succeeded
# █▄▀ █▀▀ █▄█ █▄▄ █ █▄░█ █▀▄ █ █▄░█ █▀▀ █▀
# █░█ ██▄ ░█░ █▄█ █ █░▀█ █▄▀ █ █░▀█ █▄█ ▄█

#bind = $CONTROL ALT SHIFT, T, exec, ~/.config/hypr/scripts/testrunner.sh # test runner
bind = $CONTROL ALT SHIFT, R, pass, ^(com\.obsproject\.Studio)$ # start/stop obs screen recording // install obs flatpak
bind = $mainMod SHIFT, G, exec, pkill -x rofi || ~/.config/hypr/scripts/gamelauncher.sh 3 # steam game launcher // install steam
bind = $mainMod, period, exec, emote # open emoji selector // install emote
bind = $mainMod ALT, XF86MonBrightnessDown, exec, hyprshade on blue-light-filter # enable blue light filter // install hyprshade
bind = $mainMod ALT, XF86MonBrightnessUp, exec, hyprshade off # disable blue light filter // install hyprshade


#█░█ █▀▄▀█
#▀▄▀ █░▀░█

bind = CTRL Alt_L, V, submap, passthrough
submap = passthrough
bind = CTRL Alt_L, V, submap, reset
submap = reset



Config File: /home/noah/.config/hypr/nvidia.conf: Read Succeeded

# █▄░█ █░█ █ █▀▄ █ ▄▀█
# █░▀█ ▀▄▀ █ █▄▀ █ █▀█


# See https://wiki.hyprland.org/Nvidia/ for more

#env = GBM_BACKEND,nvidia-drm
env = LIBVA_DRIVER_NAME,nvidia
env = __GLX_VENDOR_LIBRARY_NAME,nvidia
env = __GL_VRR_ALLOWED,1
env = WLR_NO_HARDWARE_CURSORS,1
env = WLR_DRM_NO_ATOMIC,1


======Config-End========
```

### Description

Hello,

Since the last update, 0.41 or 0.41.1 (I'm not sure but I think 0.41.1) , the scroll of my mouse has become too powerful. The `scroll_factor` variable in input doesn't change a thing. But the scroll of my touchpad is still the same and still good.
The mouse is a logitech mx master 3 in bluetooth

### How to reproduce

Update your hyprland, have a mouse

**Comment by vaxerski** [comment]
use scroll_factor https://wiki.hyprland.org/Configuring/Variables/

**Comment by noahfraiture** [comment]
> use scroll_factor https://wiki.hyprland.org/Configuring/Variables/

Like said in the description this doesn't work

**Comment by vaxerski** [comment]
can you bisect?

@Agent00Ming you tinkered with scroll recently

**Comment by noahfraiture** [comment]
I plan to make a whole reset in a week, I'll tell you after that and bisect if the problem persist

**Comment by Agent00Ming** [comment]
It's a side effect of some 'smooth-scrolling' mice sending way too many events and not many apps supporting and handling the high-res scrolling events I believe. There was a user in the discord that did a thing to disable their mouse's smooth scrolling mode

**Comment by IgnIVertiKalCaD** [comment]
I have for 1 scroll down, counts as 5 scrolls down and no **scroll_factory does not** help with this problem. This behavior I observed only in chrome on versions BEFORE 0.40.0 inclusive. How can this be fixed?

**Comment by noahfraiture** [comment]
Is it possible to open this issue ? The problem persist.  A scroll factor of 0 works but any other values don't

**Comment by vaxerski** [comment]
we'll try to fix it some way in or after #6608
